test-harness/specs/security-scheme-validation-AC-8.oas2.txt

```diff
@@ -0,0 +1,29 @@
+====test====
+Resource is returned when an api-key is provided as a header.
```

Comment:
Assumed: no validation needed, just api key presence is enough for the security to pass.
Maybe the api-key should actually be validated somehow?